src/opencmiss/zincwidgets/glyphchooserwidget.py

```diff
@@ -1,17 +1,22 @@
-"""
-Zinc Glyph Chooser Widget
+'''
```

Comment:
All `'''` should be `"""` but I think I would actually prefer this to be done as a separate PR.